roles/openshift_cloud_provider/tasks/aws.yml

```diff
@@ -0,0 +1,18 @@
+- name: Set AWS provider facts
+  openshift_facts:
+    role: aws
+    local_facts:
+      access_key: "{{ openshift_cloud_provider_aws_access_key | default(None) }}"
+      secret_key: "{{ openshift_cloud_provider_aws_secret_key | default(None) }}"
```

Comment:
I considered this but that would end up being `openshift.cloud.provider.aws.access.key` which is fine by me, nobody is going to see it...